### PR TITLE
Home page 02

### PR DESCRIPTION
**Pull request by ValentinCop**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

File: frontend/src/pages/Home/style.jsx [hunk 1]
```diff
@@ -2,27 +2,117 @@ import styled from "styled-components";
...
+    display: none;
   }
 
-  .banner p {
-    color: white;
-    width: 80vw;
-    font-size: 1.5rem;
-    font-family: var(--main-font);
-    text-align: center;
+  .about {
```
Comment (LuigiMenez) [inline frontend/src/pages/Home/style.jsx]: Possibility to make `.about, .connection p{display: none;}`

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Modified style

**Review by ghost — APPROVED** [review]

File: frontend/src/pages/Home/style.jsx (hunk 1 above)
Comment (ghost) [reply frontend/src/pages/Home/style.jsx]: Done

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**